video/out/vo_gpu_next.c

```diff
@@ -821,6 +821,9 @@ static void apply_target_options(struct priv *p, struct pl_frame *target)
         target->color.transfer = mp_trc_to_pl(opts->target_trc);
     if (opts->target_peak)
         target->color.hdr.max_luma = opts->target_peak;
+    // If swapchain returned a value use this, override is used in hint
+    if (opts->target_peak && !target->color.hdr.max_luma)
+        target->color.hdr.max_luma = opts->target_peak;
```

Comment:
This seems logically redundant with lines 822-823?

Reply:
Of course... Fixed